### ISSUE TITLE
Add Schöffengericht module

### ISSUE THREAD

**Issue by barnslig** [nmeum/marvin#5]
:horse: :100: 

This bot needs a way to kick(-bann) people joining our goddamn IRC channel. :rage:  System should be like this: 

```
hanspeter (~doedelmann@hackfresse.uberweltraum.de) has joined #secretsociety
dieter | !gegen hanspeter
Notice(PedroFuego): 1 vote against hanspeter, two left until law enforcement
dieter | RAUS HIER, ABER SCHNELL!
gollum | hanspeter: bist du süß?
gollum | !gegen hanspeter
Notice(PedroFuego) 2 votes against hanspeter, one left until law enforcement
fofo   | !fuer hanspeter
Notice(PedroFuego) 2 votes against hanspeter, 1 vote for hanspeter, two left until law enforcement
dieter | fofo: DEIN ERNST
…
…
Notice(PedroFuego) The court paused the trial. Votes for hanspeter resetted
```

tl;dr :books:: Someone not having `+o` hanging around can be kicked out by fellow channel members by voting against or for the suspicious person. If not enough contra-votes are collected within a specific time, e.g. 5 minutes :clock5:, the votes counter gets resetted. :tada:

Collective mobbing has newer been that much fun :exclamation:

**Comment by sternenseemann** [comment]
Please rename the commands! It makes me sick in my stomach :mask: because it reminds me of #nodrama.de :rage: :rage: :rage:

**Comment by Teufelchen1** [comment]
I suggest to rename :writing_hand: the commands:
!court - Shows all running courts
!like - up vote someone
!hate - down vote someone
are you in :cupid: with that?
